CHANGELOG.md

```diff
@@ -135,21 +113,33 @@
 * [BUGFIX] Fix reporting of tracing spans from PromQL engine. #2707
 * [BUGFIX] Apply relabel and drop_label rules before forwarding rules in the distributor. #2703
 * [BUGFIX] Distributor: Register `cortex_discarded_requests_total` metric, which previously was not registered and therefore not exported. #2712
+* [BUGFIX] Ruler: fix not restoring alerts' state at startup. #2648
+* [BUGFIX] Ingester: Fix disk filling up after restarting ingesters with out-of-order support disabled while it was enabled before. #2799
+* [BUGFIX] Memberlist: retry joining memberlist cluster on startup when no nodes are resolved. #2837
+* [BUGFIX] Query-frontend: fix incorrect mapping of http status codes 413 to 500 when request is too large. #2819
+* [BUGFIX] Alertmanager: revert upstream alertmananger to v0.24.0 to fix panic when unmarshalling email headers #2924 #2925
+* [BUGFIX] Fix sanity check done on configured filesystem directories when running Alertmanager in microservices mode. #2947
```

Comment:
This entry should be added to:

> * [ENHANCEMENT] Add sanity check at startup to ensure the configured filesystem directories don't overlap for different components. #2828

Reply:
Is there a process to finding these? Or do you just know? I think I'm missing a step here.

Reply:
https://github.com/grafana/mimir/pull/2995/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR98 That's added here

Reply:
Oh I see